.github/workflows/node.js.yml

```
@@ -0,0 +1,16 @@
+name: Node.js CI
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ["7.6.0", 8.x, 10.x, 12.x, 14.x, 16.x, 18.x]
+    steps:
+    - uses: actions/checkout@v3
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+    - run: npm install
```

https://docs.npmjs.com/cli/v8/commands/npm-ci
```suggestion
    - run: npm ci
```

---

We can't use `npm ci` because we don't commit `package-lock.json`

---

ah, right :+1: